benchmarks/benchmark_serving.py

```diff
@@ -784,6 +860,20 @@ def main(args: argparse.Namespace):
         " context. The length range of context in a random "
         " request is [random-prefix-len, "
         " random-prefix-len + random-prefix-len * random-range-ratio).")
+    parser.add_argument(
+        "--hf-output-len",
+        type=int,
+        default=128,
```

Comment:
Why do we set a default override here unlike `None` similar to what we did with ShareGPT?